aws-api/build.gradle

```diff
@@ -17,6 +17,8 @@ apply plugin: 'com.android.library'
 apply from: rootProject.file("configuration/checkstyle.gradle")
 apply from: rootProject.file("configuration/publishing.gradle")
 
+ext.instrumentedTestAmplifyBackend = "ApiInstrumentedTests"
```

Comment:
Can this live in the root `build.gradle`, so that you don't have to add it to each module?